.env.example

```diff
@@ -1,26 +1,30 @@
-# bancho.py supports both INET and UNIX sockets
-# INET: provide an IP for SERVER_ADDR, and a port for SERVER_PORT
-# UNIX: provide a filesystem path for SERVER_ADDR, and no SERVER_PORT
-SERVER_ADDR=/tmp/bancho.sock
-SERVER_PORT=
+# for nginx reverse proxy to work through the containers, the bancho
+# server must expose itself on port 80 to be accessed on http://bancho.
+SERVER_ADDR=0.0.0.0
+SERVER_PORT=80
```

Comment:
if these are constant, no need to include them here

just the external host (which may vary) details are important

Reply:
ah i see, it matters outside of docker setup..

Reply:
can this be resolved?